src/vs/workbench/contrib/preferences/browser/settingsWidgets.ts

```diff
@@ -831,7 +831,9 @@ export class ObjectSettingWidget extends AbstractListSettingWidget<IObjectDataIt
 			? item.key.options.find(({ value }) => item.key.data === value)?.description
 			: undefined;
 
-		return enumDescription ?? localize('objectPairHintLabel', "The key `{0}` maps to `{1}`", item.key.data, item.value.data);
+		return isDefined(enumDescription)
+			? `${enumDescription} Currently set to ${item.value.data}.`
```

Comment:
Add a period after the description

Reply:
Should this include the actual key?

Reply:
The screen reader says the row out in the following fashion:

"<key> <value>" "<row.title>"

So, for a row with key=*.ts, and value=typescript:

"*.ts typescript. The property *.ts is set to typescript"

For keys that have a description, it will say:

"*.ts typescript. <Description>. Currently set to typescript."

which is why I don't think that we should include keys here.